Source/Core/VideoBackends/Software/Tev.h

```diff
@@ -22,33 +102,97 @@ class Tev
   };
 
   // color order: ABGR
-  s16 Reg[4][4];
-  s16 KonstantColors[4][4];
-  s16 TexColor[4];
-  s16 RasColor[4];
-  s16 StageKonst[4];
-  s16 Zero16[4];
-
-  s16 FixedConstants[9];
+  TevColor Reg[4];
```

Comment:
Is there a reason this isn't  using a `std::array`?

Reply:
It was changed to a `Common::EnumMap` in a later commit.  I think I left it as a raw array as a reminder to myself to change it later, but I don't think there's anything preventing it from being a `std::array` temporarily.

Reply:
Ah I missed it was change.  Sorry, that's fine.

Reply:
I've changed it to an std::array in that commit anyways, since it's already being touched there (it's still changed to an EnumMap in the later commit).